sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
```

Comment:
See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=sdk-generation-from-updated-spec
```
6. Update the following information in AssemblyInfo.cs
  - AssemblyFileVersion (Must for new nuget package)
  - AssemblyVersion
```

Reply:
All track 2 and dataplane sdk don't have these properties